UITest/UITestSample.cs

```diff
@@ -43,67 +45,82 @@ public void BeforeEachTest()
         public void FacebookLogin()
         {
             // Start app only once, so that further test can continue
-            app.Tap(c => c.Marked("button1"));
+            app.Tap(c => c.Marked("loginButton"));
             Thread.Sleep(3000);
             app.Screenshot("Login");
             app.TapCoordinates(549, 900);
             app.EnterText("cvnewggbsc_1487629189@tfbnw.net");
-            // app.EnterText(c => c.Marked("NoResourceEntry-42"), "cvnewggbsc_1487629189@tfbnw.net");
             app.ScrollDownTo(c => c.Css("input#u_0_2"));
             app.EnterText(c => c.Css("input#u_0_2"), "mi-390");
-            app.ScrollToVerticalStart();
-            app.TapCoordinates(549, 1300);
-            app.WaitForElement(c => c.Marked("radioButtonNonSmoker"));
+            app.PressEnter();
         }
+
         [Test]
         public void SelectPreferences()
         {
-            app.WaitForElement(c => c.Marked("button1"));
+            app.WaitForElement(c => c.Marked("loginButton"));
             app.Invoke("StartActivityOne");
             app.Tap(c => c.Marked("radioButtonNonSmoker"));
             app.Tap(c => c.Marked("radioButtonNoLuggage"));
             app.Tap(c => c.Marked("radioButtonNoHandicap"));
             app.Tap(c => c.Marked("radioButtonNoPet"));
             app.Tap(c => c.Marked("text1"));
```

Comment:
We should change this id to make sure it's as specific as possible, and so we don't accidentally have more than 1 element with the same id